.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: Build
+on:
+  - push
+  - pull_request
+  - workflow_dispatch
+
+jobs:
+  build-python:
+    uses: goes-funky/workflows/.github/workflows/build-python.yaml@make-ssh-private-key-optional-python
```

Comment:
Also, please change this back to main/master. Otherwise it will break as soon as my PR is merged.